akeyless_gateway/README.md

```diff
@@ -0,0 +1,107 @@
+
+```
+metrics:
+  enabled: true  
+  config: |
+    exporters:    
+      datadog:
+        api:
+          key: "<Your Datadog API key>"
+          site: <Your Datadog server site>         
+    service:
+      pipelines:
+        metrics:
+          exporters: [datadog]
+```
+
+2. If you have not yet deployed the Gateway, continue with your installation as usual and run the following command when ready to deploy:
+
+```
+helm install <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+3. If you are updating an existing Gateway on Kubernetes, run the following commands to update:
+
+```
+helm upgrade <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+#### For a standalone Gateway running on Docker
+
+To configure the Akeyless Gateway integration on a [Standalone Gateway](https://docs.akeyless.io/docs/install-and-configure-the-gateway):
+
+1. Create a local file called `otel-config.yaml` with the below configuration. Set the relevant API Key of your Datadog server, and the relevant [Datadog site][1] (i.e. `app.datadoghq.com`).
+
+```
+exporters:
+  datadog:
+    api:
+      key: "<Your Datadog API key>"
+      site: <Your Datadog server site>
+service:
+  pipelines:
+    metrics:
+      exporters: [datadog]
+```
+
+2. If you have not yet deployed the Gateway, run the below command to spin up your Akeyless Gateway with the `ENABLE_METRICS=true` variable and mounting the `otel-config.yaml` file:
+
+```
+docker run -d -p 8000:8000 -p 8200:8200 -p 18888:18888 -p 8080:8080 -p 8081:8081 -p 5696:5696 -e ENABLE_METRICS="true" -v $PWD/otel-config.yaml:/akeyless/otel-config.yaml --name <your-gateway-name> akeyless/base:latest-akeyless
+```
+3. If you are updating an existing Gateway, use the same `Admin Access ID` and `Cluster Name` for the updated Gateway in order to retrieve the latest settings and data from the previously removed Docker instance:
+
+```
+docker run -d -p 8000:8000 -p 8200:8200 -p 18888:18888 -p 8080:8080 -p 8081:8081 -p 5696:5696 -e ADMIN_ACCESS_ID="p-xxxxxx" -e ADMIN_ACCESS_KEY="62Hu...xxx....qlg=" -e ENABLE_METRICS="true" -v $PWD/otel-config.yaml:/akeyless/otel-config.yaml --name <your-gateway-name> akeyless/base:latest-akeyless
+```
+
+### Validation
+
+Upon successful setup of the Gateway, go to the Metrics page on the Datadog platform, and filter the Akeyless metrics on the summary page.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv](/akeyless_gateway/metadata.csv) for a list of metrics provided by this integration.
+
+### Service Checks
+
+Akeyless Gateway does not include any service checks.
+
+### Events
+
+Akeyless Gateway does not include any events.
+
+## Support
+
+Need help? Contact [Akeyless Support](mailto:support@akeyless.io).
+
+
+[1]: https://docs.datadoghq.com/getting_started/site/
```

Comment:
Remember to replace [5] with a link to the "Metrics page on the Datadog Site that shows Akeyless metrics on the summary page".

```suggestion
[1]: https://docs.datadoghq.com/getting_started/site/
[2]: https://docs.akeyless.io/docs/api-gw
[3]: https://docs.akeyless.io/docs/gateway-k8s
[4]: https://docs.akeyless.io/docs/install-and-configure-the-gateway
[5]: TBD
[6]: https://github.com/DataDog/integrations-extras/blob/master/akeyless_gateway/metadata.csv
[7]: mailto:support@akeyless.io
```